components/cadastre-etalab.js

```diff
@@ -43,14 +43,14 @@ const CadastreEtalab = () => (
     <Section>
       <div className='main'>
         <h4>Raison d’être de ce jeu de données</h4>
-        <p>Le <Link href='/datasets/plan-cadastral-informatise'>Plan Cadastral Informatisé</Link> est diffusé sous deux formats experts rendant sa réutilisation difficile en dehors du contexte des applications métiers.<br />
+        <p>Le <Link href='/datasets/plan-cadastral-informatise'><a>Plan Cadastral Informatisé</a></Link> est diffusé sous deux formats experts rendant sa réutilisation difficile en dehors du contexte des applications métiers.<br />
```

Comment:
Pourquoi ?

Reply:
`Warning: You're using a string directly inside <Link>. This usage has been deprecated. Please add an <a> tag as child of <Link>`